### PR TITLE
comic-mono: init at 2020-12-28

### PR DESCRIPTION
**Pull request by an-empty-string**
###### Motivation for this change

Comic Mono is a nice programming font that I use (currently building from a private overlay). There is no open packaging request issue (although I'd open one if I couldn't package it).

The PR also adds myself as a maintainer so the package will build.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable: (none are applicable, this package doesn't add anything testable)
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage) (N/A)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`) (N/A, no binary files)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes) (N/A)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/comic-mono/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  owner = "dtinth";
+  repo = "comic-mono-font";
+  rev = "9a96d04cdd2919964169192e7d9de5012ef66de4";
+
+  postFetch = ''
+    mkdir -p $out/share/fonts
+    tar -z -f $downloadedFile --wildcards -x \*.ttf --one-top-level=$out/share/fonts
+
+    mkdir -p $out/etc/fonts/conf.d
+    cp ${./comic-mono-weight.conf} $out/etc/fonts/conf.d/30-comic-mono.conf
```
Comment (legendofmiracles) [inline pkgs/data/fonts/comic-mono/default.nix:17]: ${} already copies it in the store, so we can link to save a few extra bytes

```suggestion
    ln -s ${./comic-mono-weight.conf} $out/etc/fonts/conf.d/30-comic-mono.conf
```

Comment (an-empty-string) [reply pkgs/data/fonts/comic-mono/default.nix]: Thanks, fixed!

**Review by totoroot — APPROVED** [review]
This looks good to me and I'd like to see your contribution merged. Too bad the PR got buried.
@SuperSandro2000, could you please review and merge?

**Comment by totoroot** [comment]
@an-empty-string It seems as if the commit adding you as a maintainer must be resolved.
Could you please rebase from nixpkgs/master? I use comic-mono on NixOS too, so if you don't mind can you please add me as a package maintainer as well?

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -13432,4 +13432,10 @@
     github = "PhilippWoelfel";
     githubId = 19400064;
   };
+  an-empty-string = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:13435]: Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

**Comment by an-empty-string** [comment]
@totoroot glad to see I'm not the only comic mono user here :) I added you as a maintainer.

@SuperSandro2000 thanks, I've rebased against master and put my maintainer entry in the right spot alphabetically, let me know if you need any other changes!
